### DISCUSSION TITLE
SCRAM SHA 512 Password Special Characters

### DISCUSSION THREAD

**Discussion by madhusnagaraj** [strimzi/strimzi-kafka-operator#5368] [Q&A]
Is there a restriction in defining a password with special characters while using SASL SCRAM ?

**Comment by scholzj** [comment]
In Strimzi, we generate the passwords with alphanumerical characters. I think there will be some limitations when it comes to configuring the password in Java clients in the JAAS configuration. 

@im-konge Didn't you or someone run into some restrictions in system tests?

**Reply by Frawless** [reply]
I am not aware of any issues with passwords on clients side, but maybe we just didn't hit it.
